client/core/types.go

```diff
@@ -151,6 +151,7 @@ type Market struct {
 	Orders          []*Order `json:"orders"`
 	pendingSuspend  *time.Timer
 	suspended       bool
+	subscribed      bool
```

Comment:
Using this new field is unnecessary. If there is an entry in the `(*dexConnection).books` map, the client is subscribed.